core/src/test/java/org/infinispan/stream/DistributedStreamIteratorTest.java

```diff
@@ -365,22 +365,20 @@ public void testLocallyForcedStream() {
       assertEquals(values.size(), count);
    }
 
-   public void testIteratorClosedProperlyOnClose() {
-      Cache<Object, String> cache0 = cache(0, CACHE_NAME);
-      Cache<Object, String> cache1 = cache(1, CACHE_NAME);
+   private void testIteratorClosedProperlyOnClose(Cache<Object, String> dataOwnerCache, Cache<Object, String> iteratorCache) {
       Cache<Object, String> cache2 = cache(2, CACHE_NAME);
 
       // We have to insert over the buffer size default - which iterator uses
       for (int i = 0; i < Flowable.bufferSize() + 2; ++i) {
          // We insert 2 values into caches where we aren't the owner (they have to be in same node or else iterator
          // will finish early)
-         cache0.put(magicKey(cache1, cache2), "not-local");
+         dataOwnerCache.put(magicKey(dataOwnerCache, cache2), "value");
```

Comment:
The comment above is no longer correct, since now "we" can be the data owner cache

Reply:
Sure, I didn't even look at the comments :)